inc/class-wpseo-replace-vars.php

```diff
@@ -111,8 +111,12 @@ public static function register_replacement( $var, $replace_function, $type = 'a
 			}
 			elseif ( ! method_exists( __CLASS__, 'retrieve_' . $var ) ) {
 				if ( ! isset( self::$external_replacements[ $var ] ) ) {
+					if ( empty( $var ) ) {
```

Comment:
This is my mistake, we should have moved this `if` statement between line 112 and 113